server.js

```diff
@@ -13,30 +13,32 @@ const app = express();
 // Use EJS for HTML templating.
 app.set('view engine', 'ejs');
 
-app.use(express.static(__dirname + "/static"));
+app.use(express.static(__dirname + '/static'));
```

Comment:
My linter went a little craz. If you'd like to review the actual code changes, I'd suggest hiding whitespace changes at the top here:
<img width="641" alt="Screenshot 2024-10-21 at 11 28 22 AM" src="https://github.com/user-attachments/assets/51d277f9-2d69-4099-a6f0-e2f5ff86c650">